### PR TITLE
Add Latex Note for lecture 2 and 11

### PR DESCRIPTION
**Pull request by simon-mo**
This pull request adds two typed notes to the website, along with the
source files the assets folder. `lectures.yml` is modified to include
the link to the pdfs.

- Simon Mo

### CODE REVIEW

File: _data/lectures.yml [hunk 1]
```diff
@@ -446,7 +450,7 @@
   topic: Holiday!!
 
 - date: 11/28/2017
-  topic: Big Data Ecosystemj
+  topic: Big Data Ecosystem
```
Comment (SamLau95) [inline _data/lectures.yml:453]: Thanks for the catch :)

File: assets/notes/11.tex [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  \hline
+  chance & $\frac{2}{5}$ &$\frac{2}{5}$&$\frac{1}{5}$ \\
+\end{tabular}
+\end{center}
+
+Since the distribution are the same, the expected value, variance, and $\SD$ will be the same. \begin{equation*}
+	\E(X_2) = \E(X_1) = 88 \quad \quad \Var(X_2) = \Var(X_1)
+\end{equation*}
+
+Next, we consider the average $\bar{X} = \frac{X_1 + X_2}{2}$. Note this is a random variable too. Sometimes the average is $80$, when both draws are $80$ or $86$ when one draw is $80$ and the other is $92$. What are teh values $\bar{X}$ can take on? \marginnote{\twolines Where dose the chance, say $\frac{8}{25}$ comes from? The denominator is $25$ because there are $5 \times 5 =25$ possible pairs $(X_1,X_2)$, $8$ of these $25$ have an average of $86$. These are $
```
Comment (SamLau95) [inline assets/notes/11.tex]: Typo: `teh` -> `the`, `dose` -> `does`

File: assets/notes/11.tex [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+	\E(X_1 + X_2) = \sum_{j=1}^m\sum_{k=1}^m (v_j + v_k) \P(X_1 = v_j, X_2 = v_k)
+\end{equation*}
+
+How do we proceed? We need to understand how random variable can vary together. 
+
+\section{Aside: Joint Distribution}
+$\P(X_1 = v_j, X_2 = v_k) = $ chance that $X_1$ is $v_j$ and $X_2$ is $v_k$. With two random variable we have joint probability: 
+
+
+\includegraphics[scale=0.3]{joint.png}\\
```
Comment (SamLau95) [inline assets/notes/11.tex:175]: Nice images!

**Review by SamLau95 — CHANGES_REQUESTED** [review]
Thanks so much for latexing the notes! Just got a couple typos but this looks ready to go after they're fixed.

File: assets/notes/11.tex (hunk 2 above)
Comment (simon-mo) [reply assets/notes/11.tex]: Fixed! Thanks Sam

**Comment by simon-mo** [comment]
Hi @SamLau95 Typos has been fixed! You can merge it now. Thank you!

**Review by SamLau95 — APPROVED** [review]

**Comment by SamLau95** [comment]
Thanks @simon-mo !

**Pull request merged**